x-pack/plugins/task_manager/server/integration_tests/removed_types.test.ts

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { v4 as uuidV4 } from 'uuid';
+import { ElasticsearchClient } from '@kbn/core/server';
+import { TaskManagerPlugin, TaskManagerStartContract } from '../plugin';
+import { injectTask, retry, setupTestServers } from './lib';
+import { TestElasticsearchUtils, TestKibanaUtils } from '@kbn/core-test-helpers-kbn-server';
+import { ConcreteTaskInstance, TaskStatus } from '../task';
+import { CreateWorkloadAggregatorOpts } from '../monitoring/workload_statistics';
+
+const taskManagerStartSpy = jest.spyOn(TaskManagerPlugin.prototype, 'start');
+
+const { createWorkloadAggregator: createWorkloadAggregatorMock } = jest.requireMock(
+  '../monitoring/workload_statistics'
+);
+jest.mock('../monitoring/workload_statistics', () => {
+  const actual = jest.requireActual('../monitoring/workload_statistics');
+  return {
+    ...actual,
+    createWorkloadAggregator: jest.fn().mockImplementation((opts) => {
+      return new actual.createWorkloadAggregator(opts);
+    }),
+  };
+});
+
+describe('unrecognized task types', () => {
+  let esServer: TestElasticsearchUtils;
+  let kibanaServer: TestKibanaUtils;
+  let taskManagerPlugin: TaskManagerStartContract;
+  let createWorkloadAggregatorOpts: CreateWorkloadAggregatorOpts;
+
+  const taskIdsToRemove: string[] = [];
+
+  beforeAll(async () => {
+    const setupResult = await setupTestServers({
+      xpack: {
+        task_manager: {
+          claim_strategy: `update_by_query`,
```

Comment:
hmmm ... should we test both claimers?  I think for the test this is ultimately making, it's not needed.

But I was curious about hard-coding the `claim_strategy`, and  I did notice a lot of usage of `claim_strategy: 'default'` in the code (search on `claim_strategy`), which is presumably logging an error once, and then returning "the default".  :-)

https://github.com/elastic/kibana/blob/c28af871d2fa4d8e9f0edd0eafad4d10669a62f5/x-pack/plugins/task_manager/server/task_claimers/index.ts#L52-L65

So, thinking maybe change this to `default` - if testing with one is fine.  That way it'll track the default we use, when we switch the default to mget.

Reply:
I guess I can remove the hard-coded claimer and it will just use whatever the default is, whenever we switch it, it'll test the default? That makes sense to me.

I can create a followup issue for the `default` claim strategies. Probably an oversight from when we switched from `default` to `update_by_query`

Reply:
Updated all the `claim_strategy: 'default'` to `update_by_query` in this commit: https://github.com/elastic/kibana/pull/193479/commits/a58040cfc86c0a6a5c68c1ba73a3954014a3bea5

Reply:
Removed the hard-coded claim strategy in the jest integration test in this commit: https://github.com/elastic/kibana/pull/193479/commits/f463ff7d8d8727d551cf2319cf011ee66c62c8b0